scripts/generate-rule-files/cli.ts

```diff
@@ -0,0 +1,9 @@
+import { run } from '.';
+
+const selectedPlugins = process.argv
+  .slice(2)
+  .find((arg) => arg.startsWith('--plugins='))
+  ?.replace('--plugins=', '')
+  .split(',');
+
+run({ plugins: selectedPlugins }).catch((err) => console.error(err));
```

Comment:
Can we use top level await?

Reply:
Would require to move to ESM. I don't think it's worth the hassle right now

But if you guys think it's necessary, I can take a look !

Reply:
For now the project is not written in ESM, so lets do this in another PR when needed